src/goto-programs/interpreter_evaluate.cpp

```diff
@@ -885,42 +885,11 @@ void interpretert::evaluate(
     }
     return;
   }
-  else if(expr.id()==ID_byte_extract_little_endian ||
-          expr.id()==ID_byte_extract_big_endian)
-  {
-    if(expr.operands().size()!=2)
-      throw "byte_extract should have two operands";
-    mp_vectort extract_offset;
-    evaluate(expr.op1(), extract_offset);
-    mp_vectort extract_from;
-    evaluate(expr.op0(), extract_from);
-    if(extract_offset.size()==1 && extract_from.size()!=0)
-    {
-      const typet &target_type=expr.type();
-      mp_integer memory_offset;
-      // If memory offset is found (which should normally be the case)
-      // extract the corresponding data from the appropriate memory location
-      if(!byte_offset_to_memory_offset(
```

Comment:
You remove this check. No idea what it does, but it should probably be removed in a separate commit to make the move clearer.